strongbox-resources/strongbox-storage-api-resources/src/main/resources/etc/conf/strongbox.yaml

```diff
@@ -54,6 +54,8 @@ configuration:
           repositoryConfiguration:
             type: Maven 2
             indexingEnabled: true
+            downloadRemoteMavenIndexCronExpression: "0 0 2 * * ?"
+            rebuildMavenIndexesCronExpression: "0 0 2 * * ?"
```

Comment:
actually we have 3 cron jobs 

`RebuildMavenIndexesCronJob` - works for hosted repository
`DownloadRemoteMavenIndexCronJob` - works for proxy repository
`MergeMavenGroupRepositoryIndexCronJob` - works for group repository

there should be one `***CronExpression` per repository depending on the repository type